app/views/listings/show.html.erb

```diff
@@ -1,9 +1,9 @@
 <div class="container">
-  <h2><%= @Listing.title %></h2>
-  <p>Description : <%= @Listing.description %></p>
-  <p>Vendor : <%= @Listing.user %></p>
-  <p>Location : <%= @Listing.location %></p>
-  <p>Activity duration : <%= @Listing.duration %>hours</p>
+  <h2><%= @listing.title %></h2>
+  <p>Description : <%= @listing.description %></p>
+  <p>Vendor : <%= @listing.user %></p>
```

Comment:
@listing.user here will display the instance of "user" which is associated with the listing (Which will include ALL of the info about the user including id, email, password, first_name, last_name etc) - here I think we need to show vendor_name which I believe would be @listing.user.vendor_name (but test it!)